### DISCUSSION TITLE
rpzPrimary url support

### DISCUSSION THREAD

**Discussion by 24fpsDaVinci** [PowerDNS/pdns#14738] [Q&A]
right now pdns-recursor only supports getting rpz through ip address:

`rpzPrimary("192.0.2.3", "file.rpz")`

how can we download rpz files hosted by websites? eg:

`rpzPrimary("big.oisd.nl/rpz", "file.rpz")`

**Comment by omoerbeek** [comment]
`rpzPrimary` can only download RPZs using DNS (A/I)XFR at the moment, which is much more efficient because it only downloads changes most of the time.

What you can do as a workaround is create a cronjob to download the file, and load it with `rpzFile`.
